Sources/MongoSwift/MongoCollection.swift

```diff
@@ -12,6 +12,33 @@ public struct DropCollectionOptions: Codable {
     }
 }
 
+/// Options to use when renaming a collection.
```

Comment:
Given the difference in names between the options and the method they're used for, can we add another sentence to this docstring mentioning the options will be used for `MongoCollection.renamed`? If you include it in backticks `\`\`` like that, the docs generator should automatically linkify that text when it gets rendered.